docs/fundamentals/dashboard/usage-telemetry.md

```diff
@@ -0,0 +1,49 @@
+---
+title: .NET Aspire dashboard telemetry
+description: Learn about what telemetry the .NET Aspire dashboard sends and how to opt out.
+ms.date: 03/05/2025
+---
+
+# .NET Aspire dashboard telemetry
+
+The Aspire dashboard includes a telemetry feature that collects usage data when the dashboard is launched through Visual Studio or Visual Studio Code as part of a running Aspire application. This information is sent to Microsoft to help the Aspire team understand how the dashboard is used and help improve the product. Exception information is also sent when unhandled exceptions occur in the dashboard.
+
+## Scope
+
+.NET Aspire dashboard usage telemetry is supported when using Visual Studio or Visual Studio Code to run an Aspire application containing a dashboard resource.
+
+Telemetry is collected only when the Aspire dashboard is open in the browser and the instance of Visual Studio or Visual Studio Code has not opted out of telemetry collection.
+
+## How to opt out
+
+.NET Aspire dashboard telemetry is enabled by default for `Aspire >= 9.2` when launched through Visual Studio `>= 17.14` or C# Dev Kit `>= [VSC RELEASE VERSION]`.
+
+To opt out, you may either:
+
+- Set the `DOTNET_DASHBOARD_ENABLE_TELEMETRY` environment variable to `false`. This will apply to all users accessing the Aspire dashboard.
```

Comment:
Could a user alternatively set an `AppContext` switch?

```cs
AppContext.SetSwitch("DOTNET_DASHBOARD_ENABLE_TELEMETRY", true);
```

Reply:
Not currently. That pattern is not currently supported for any dashboard configuration option, so it needs further discussion. @davidfowl how do you feel about supporting switches?